Firebase/android/core/src/main/java/com/novoda/bonfire/channel/presenter/ChannelsPresenter.java

```diff
@@ -48,6 +59,7 @@ public void call(Authentication authentication) {
                     }
                 })
                 .flatMap(channelsForUser())
+                .map(sortIfConfigured())
```

Comment:
Of the config
On 13 Jun 2016 15:51, "Benjamin AUGUSTIN" <notifications@github.com> wrote:

> In
> Firebase/android/core/src/main/java/com/novoda/bonfire/channel/presenter/ChannelsPresenter.java
> <https://github.com/novoda/spikes/pull/91#discussion_r66794376>:
>
> > @@ -48,6 +59,7 @@ public void call(Authentication authentication) {
> >                      }
> >                  })
> >                  .flatMap(channelsForUser())
> > +                .map(sortIfConfigured())
>
> The new values of the config or of the data ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/novoda/spikes/pull/91/files/9a6763b08f84cfdd24cd99679dc6ab7dac78fcd9#r66794376>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYcWdqIQqdflOt5Q6naIx78yQTaj34vks5qLWB-gaJpZM4I0MpW>
> .
>